pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, fetchzip, SDL, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  pname = "hex-a-hop";
+  version = "1.1.0";
+
+  src = fetchzip {
+    url = "https://downloads.sourceforge.net/project/hexahop/${version}/hex-a-hop-${version}.tar.gz";
+    sha256 = "sha256-fBSvNtgNR0aNofbvoYpM1e8ww4ARlXIvrQUvJqVGLlY=";
+  };
+
+  icon = fetchurl {
+    url = "https://sources.debian.org/data/main/h/hex-a-hop/${version}%2Bgit20140926-1.1/data/hex-a-hop.png";
+    sha256 = "sha256-Vh/1wwRmC2eSD/+mk1Oqt7EX4a4k++nUAbWQD2P2hNA=";
+  };
+
+  buildInputs = [ SDL SDL_mixer SDL_ttf ];
+
+  nativeBuildInputs = [ copyDesktopItems ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  desktopItems = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    desktopName = pname;
+    comment = meta.description;
+    type = "Application";
+    categories = [ "Game" "LogicGame" ];
+    startupNotify = false;
+  };
```

Comment:
The source contains a pre-made desktop file.

Reply:
Huh? Where?

Reply:
Under `data` IIRC.

Reply:
Are you sure you're checking the same source? Find doesn't find any .desktop anywhere inside.

Reply:
https://sourceforge.net/p/hexahop/code/ci/master/tree/data/hex-a-hop.desktop

Reply:
that's master, not the 1.1.0 version

Reply:
Ah, it used to live under the debian directory apparently. Install from there then.

Reply:
Ah, interesting. Fetch it from dfa365a90be5c52ef21235a9e90a865b04da3ad4 from the source repo then and add a comment to remove it in the next version.